api/src/main/java/io/grpc/ServerCall.java

```diff
@@ -163,6 +163,11 @@ public boolean isReady() {
    * <p>Since {@link Metadata} is not thread-safe, the caller must not access (read or write) {@code
    * trailers} after this point.
    *
+   * <p>Implementation note regarding ForwardingServerCalls: This method is called after the
```

Comment:
I'd much rather not call out ForwardingServerCalls. And saying something like "this method won't be called" isn't right either as it is up to the application whether it is called or not.

I think we should have some text to emphasize that the close call is _not_ the point where the RPC is complete. Instead, that is onComplete/onCancel. We can probably add that to the paragraph above. We can also say that close is not necessary to be called if the RPC is cancelled as the call is already dead.